CHANGES.md

```diff
@@ -1,3 +1,30 @@
+## stable-2.3.1
+
+This stable release adds a number of proxy stability improvements.
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Special thanks to**: @zaharidichev and @11Takanori!
+
+**Full release notes**:
+
+* Proxy
+  * Changed the proxy's routing behavior so that, when the control plane
+    does not resolve a destination, the proxy forwards the request with minimal
```

Comment:
IMO "forwards the request to its original destination" better describes the behaviour than "forwards the request with minimal additional routing logic".

Reply:
I think "forwards the request with minimal additional routing logic" works because it implies there is more going on behind the scenes w.r.t routing and that the behavior has now been fixed.